test/NCI.OCPL.Api.Glossary.Tests/Tests/Controllers/TermsControllerTest.GetByName.cs

```diff
@@ -360,7 +360,7 @@ public async void GetByName_WithFallback_TermsPatient()
             .Returns(Task.FromResult(glossaryTerm));
 
             TermsController controller = new TermsController(NullLogger<TermsController>.Instance, termQueryService.Object);
-            GlossaryTerm gsTerm = await controller.GetByName("NotSet", AudienceType.Patient, "en", "s-phase-fraction", true);
+            GlossaryTerm gsTerm = await controller.GetByName("notset", AudienceType.Patient, "en", "s-phase-fraction", true);
```

Comment:
Leaving this as mixed-case `NotSet` would be helpful for catching problems if the controller stops handling it correctly.  Ideally, that would be detected by tests for that specific purpose, but leaving this the way it was would not have been bad.

Reply:
Removed, as passing in the title case is a good test to have (even though we test it all over this test file).